app/src/main/res/values/strings.xml

```diff
@@ -34,32 +34,32 @@
   <string name="menu_star_show">Star show</string>
   <string name="menu_unstar_show">Unstar show</string>
   <string name="menu_archive_show">Archive show</string>
-  <string name="menu_unarchive_show">Unarchive show</string>
+  <string name="menu_unarchive_show">Remove show from archive</string>
   <string name="pref_auto_refresh_period_title">Interval</string>
-  <string name="pref_auto_refresh_enabled_title">Auto-refresh Enabled</string>
-  <string name="pref_auto_refresh_wifi_only_title">Only on unmetered connexions</string>
+  <string name="pref_auto_refresh_enabled_title">Auto-refresh on</string>
+  <string name="pref_auto_refresh_wifi_only_title">Only on unlimited Wi-Fi</string>
```

Comment:
```suggestion
  <string name="pref_auto_refresh_enabled_title">Auto-refresh</string>
  <string name="pref_auto_refresh_wifi_only_title">Only on unmetered connexions</string>
```
I think the string name might be misleading because IIRC you can have a metered Wi-Fi connexion in newer Android versions or unmetered connexions that are not WiFi e.g. Bluetooth. Also, since the setting is a toggle, does it make more sense to just say "Auto-refresh"?

Reply:
My point to make was to avoid "connexions". :)

Reply:
Ahh, I see now - it's an older spelling. We can go with the changes then.